app/assets/src/components/Samples.jsx

```diff
@@ -877,50 +907,97 @@ class Samples extends React.Component {
       </div>
     );
 
-    const host_filter_tag_list = this.generateTagList(
-      "hostGenomes",
-      "selectedHostIndices",
-      "Host: ",
-      "id",
-      "name",
-      "int"
-    );
-
-    const tissue_filter_tag_list = this.generateTagList(
-      "tissueTypes",
-      "selectedTissueFilters",
-      "Sample Type: "
-    );
+    let search_tag_list;
+    if (this.admin !== 0 || this.allowedFeatures.includes("structuredSearch")) {
+      search_tag_list = this.state.searchTags.map((entry, i) => {
+        return (
+          <Label className="label-tags" size="tiny" key={`search-tag-${i}`}>
+            {entry.display}
+            <Icon
```

Comment:
Use an icon from `ui/icons` or add one yourself.
I believe for this is the `RemoveIcon`.

Reply:
ok